ReactAndroid/src/main/java/com/facebook/react/views/swiperefresh/ReactSwipeRefreshLayout.java

```diff
@@ -20,11 +20,26 @@
  */
 public class ReactSwipeRefreshLayout extends SwipeRefreshLayout {
 
+  private boolean mRefreshing = false;
+
   public ReactSwipeRefreshLayout(ReactContext reactContext) {
     super(reactContext);
   }
 
   @Override
+  public void setRefreshing(boolean refreshing) {
+    mRefreshing = refreshing;
+    // Use `post` otherwise the control won't start refreshing if refreshing is true when
+    // the component gets mounted.
+    post(new Runnable() {
```

Comment:
Not sure if it is good or bad but we have ReactSwipeRefreshLayout that posts a new Runnable with setRefreshing call while all the other Runnables are invoked from SwipeRefreshLayoutManager.
Does not look consistent and makes me confused.